client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+{
+    "common": {
+        "ok": "OK",
+        "submit": "Übernehmen",
+        "cancel": "Abbrechen",
+        "overwrite": "Überschreiben",
+        "continue": "Weiter",
+        "dismiss": "Abbrechen",
+        "prompt": "Hinweis",
+        "warning": "Warnung",
+        "attention": "Achtung!",
+        "confirm": "Bestätigen",
+        "close": "Schließen",
+        "save": "Speichern",
+        "cancelled": "Abgebrochen",
+        "enabled": "Eingeschaltet",
+        "commandOK": "Befehl erfolgreich übertragen!",
+        "commandFailed": "Es ist ein Fehler aufgetreten:",
+        "connectionError": "Verbindungsfehler",
+        "m2": "m²",
+        "hours": "Stunde",
+        "hours_plural": "Stunden",
+        "months": ["Jan","Feb","Mär","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dez"],
```

Comment:
according to DIN ISO 8601 it should be:
"months": ["Jan","Feb","Mrz","Apr","Mai","Jun","Jul","Aug","Sep","Okt","Nov","Dez"],

Reply:
Mär or Mrz this depends on who is asking... I like Mär more then Mrz.
The rest should be changed after my review of nicedevil's comments